internal/cmd/commands/targetscmd/ssh_funcs.go

```diff
@@ -130,6 +137,19 @@ func extraSshFlagsHandlingFuncImpl(c *SshCommand, _ *base.FlagSets, opts *[]targ
 		*opts = append(*opts, targets.WithSshTargetDefaultPort(uint32(port)))
 	}
 
+	switch c.flagDefaultClientPort {
+	case "":
+	case "null":
+		*opts = append(*opts, targets.DefaultSshTargetDefaultClientPort())
+	default:
+		port, err := strconv.ParseUint(c.flagDefaultClientPort, 10, 32)
```

Comment:
Very much nit but aren't port numbers limited to 16 bit unsigned integers?

Reply:
Yes, but the smallest int size in protobuf is 32 bits. We could technically do translation after generation so that our API uses 16-bit unsigned ints but we didn't in the past for other port values so I didn't here 🤷